packages/contracts-bedrock/src/L1/OptimismPortal2.sol

```diff
@@ -36,11 +36,11 @@ contract OptimismPortal2 is Initializable, ResourceMetering, ISemver {
     address internal constant SAURON = address(0xdead);
 
     /// @notice The delay between when a withdrawal transaction is proven and when it may be finalized.
-    uint256 internal immutable PROOF_MATURITY_DELAY_SECONDS;
+    uint256 public immutable PROOF_MATURITY_DELAY_SECONDS;
```

Comment:
nit: we are leaking immutable into the abi, abi should not be screaming snake case per the style guide. we should add a getter that uses snake case